renovate-config.js

```diff
@@ -4,10 +4,7 @@ module.exports = {
     repositories: [
         'gisktzh/gb3-web_ui',
         'gisktzh/gb3-search_elastic',
-        // TODO: The following three projects are not yet publicly available
-        //       Therefore, 'auto-merge' is not possible making Renovate quite noisy as all PRs must be merged manually
-        // 'gisktzh/gb3-monitoring',
-        // 'gisktzh/gb3-cms',
-        // 'gisktzh/gb3-testcases',
+        'gisktzh/gb3-cms',
+        'gisktzh/gb3-testcases',
     ],
-};
```

Comment:
Just wondering: why was the last empty line removed? It seems there is some different behavior between our IDEs

Reply:
I did it in the gui :)